### ISSUE TITLE
Migrate to package structure

### ISSUE THREAD

**Issue by gadenbuie** [gadenbuie/tidy-animated-verbs#12]
Once the dust settles on [the twitter poll](https://twitter.com/grrrck/status/1033103810158317568)...

Rename this repo. It seems that [everything will redirect](https://help.github.com/articles/renaming-a-repository/) but may need to investigate/resolve if not.

I'd like to merge #10 into a new `pkg` branch that we can use for the package development. This way we can push fixes and improvements there while leaving the current animations and documentation on the master branch. Once everything is ready to go with the package, we can merge to master.

To facilitate communication, I'd like to still use feature branches and PRs for any major feature additions or updates. Minor bugfixes can be pushed directly to `pkg`. Branch names like `pkg-add-filter-animation` are great. Let me know if there are any questions with this workflow.

I'll handle getting #10 into the `pkg` branch in this repo once it's ready.

**Comment by DavZim** [comment]
Seems like it will be tidyverbs :). 
I wonder if its smart to drop "animated" though.
Working on branches and such sounds like a smart idea!

**Comment by gadenbuie** [comment]
I agree... on the other hand, it's hard to think of a what a hypothetical parallel-universe `tidyverbs` package would do. A verb in the tidyverse should already be in a package with a better more unifying name. So I think I lean towards accepting it as a nice concise package name.

**tidyverbs**: Animate the Verbs of the Tidyverse.

or 

**tidyverbs**: Verbs of the Tidyverse, Animated.

Regardless, it does seem like there's going to be a clear winner in the poll 😄

**Comment by DavZim** [comment]
I really like the tidyexplainer name. But I guess its not really in the poll :)
Do you want to keep `pkg` open or move the package to master once you have migrated the code?

**Comment by gadenbuie** [comment]
I agree, let’s think on it for a day or two.

The name change will happen as soon as it’s settled. The plan with the pkg branch on the other hand is to treat that like the master for the package until the package is solid (more or less feature complete) and then move that branch to master.

**Comment by DavZim** [comment]
One drawback of using `tidyverbs` is that it comes alphabetically before `tidyverse`, which results in RStudio suggesting tidyverbs before tidyverse.
Only a minor annoyance though.

**Comment by gadenbuie** [comment]
That's a great point and one I hadn't thought of. For me, that convinces me against `tidyverbs`. It's a minor annoyance, but the tidyverse is a large part of many people's workflows and looking at the auto-complete text it seems we could easily create confusion when `bs` vs `se` are the two letters a user has to inspect to differentiate between the two.

<img width="188" alt="image" src="https://user-images.githubusercontent.com/5420529/44854038-8cc89580-ac35-11e8-8a62-ad9f07ddcabb.png">

I think I'm now much more on the side of `tidyexplain`. It works with the package goals and it's a somewhat "on brand" with my `RegExplain` package/addin. A tagline could be "Animated explanations of tidyverse verbs."

Btw, here are the other `tidy*` packages on CRAN as of 2018-08-30:

<details><summary>Tidy CRAN Packages</summary>

|Package                                                      |Version |second_word |first_letter |
|:------------------------------------------------------------|:-------|:-----------|:------------|
|[tidybayes](http://mjskay.github.io/tidybayes)               |1.0.1   |bayes       |b            |
|[tidyboot](https://github.com/langcog/tidyboot)              |0.1.1   |boot        |b            |
|[tidycensus](https://github.com/walkerke/tidycensus)         |0.8.1   |census      |c            |
|[tidygenomics](https://github.com/const-ae/tidygenomics)     |0.1.0   |genomics    |g            |
|[tidygraph](https://github.com/thomasp85/tidygraph)          |1.1.0   |graph       |g            |
|[tidyhydat](https://github.com/ropensci/tidyhydat)           |0.3.5   |hydat       |h            |
|[tidyimpute](https://github.com/decisionpatterns/tidyimpute) |0.1.0   |impute      |i            |
|[tidyLPA](https://jrosen48.github.io/tidyLPA/)               |0.1.3   |LPA         |L            |
|[tidymodels](https://github.com/tidymodels/tidymodels)       |0.0.1   |models      |m            |
|[tidyposterior](https://topepo.github.io/tidyposterior)      |0.0.1   |posterior   |p            |
|[tidypredict](http://tidypredict.netlify.com/)               |0.2.0   |predict     |p            |
|[tidyquant](https://github.com/business-science/tidyquant)   |0.5.5   |quant       |q            |
|[tidyr](http://tidyr.tidyverse.org)                          |0.8.1   |r           |r            |
|[tidyRSS](https://github.com/RobertMyles/tidyrss)            |1.2.6   |RSS         |R            |
|tidyselect                                                   |0.2.4   |select      |s            |
|tidystats                                                    |0.2     |stats       |s            |
|tidystringdist                                               |0.1.2   |stringdist  |s            |
|[tidytext](http://github.com/juliasilge/tidytext)            |0.1.9   |text        |t            |
|tidytidbits                                                  |0.1.0   |tidbits     |t            |
|[tidytransit](https://github.com/r-transit/tidytransit)      |0.2.3   |transit     |t            |
|[tidytree](https://github.com/GuangchuangYu/tidytree)        |0.1.9   |tree        |t            |
|[tidyverse](http://tidyverse.tidyverse.org)                  |1.2.1   |verse       |v            |
|[tidyxl](https://github.com/nacnudus/tidyxl)                 |1.0.3   |xl          |x            |
</details>
